Seed files for initial SQL database structure

ryancbrown opened the pull request:
Seed file to create local instance of SQL database. 
- ecommerce database 
- users table 
- user_profile table

ryancbrown commented:
Good call out! I left some of these null because not all countries have states or postal codes. In the event we want to just center to US at the moment we can non-null them, but thinking internationally it should be alright to not enforce these values (https://hellowahab.wordpress.com/2011/05/24/list-of-countries-without-postal-codes/). 

Open to limiting it to US and then just altering the table structure later if need be too!

megangilpin commented:
I figured you had a good reason for not requiring them.  I am fine being set up for international shipping from the get-go.  The only thing we will need to add is the country, https://faq.usps.com/s/article/International-addressing-requirements. We can eventually do validation on  State/Provence and postal code depending on what country they select.

ryancbrown commented:
:doh: Updated `user_profile` to include `country` and have also added `first_name` and `last_name` to the `users` table so we can use this for customizing emails or the page upon login rather than waiting to collect the user's name only when they purchase something.
